qualtran/bloqs/qft/qft_phase_gradient.py

```diff
@@ -18,13 +18,13 @@
 import cirq
 from numpy.typing import NDArray
 
-from qualtran import GateWithRegisters, QFxp, QUInt, Signature
+from qualtran import bloq_example, BloqDocSpec, GateWithRegisters, QFxp, QUInt, Signature
 from qualtran.bloqs.arithmetic.multiplication import PlusEqualProduct
 
 
 @attrs.frozen
 class QFTPhaseGradient(GateWithRegisters):
-    r"""QFT implemented using phase gradient trick. Uses O(n**2) T-gates for an n-bit register.
+    r"""QFT implemented using coherent addition into a phase gradient register as opposed to controlled rotations. Uses O(n**2) T-gates for an n-bit register.
```

Comment:
Thanks for the improvements. Can you re-form to get closer to the ideal format of a short, one-line summary, newline, additional information. i.e. 

```python
class QFT:
  """QFT using coherent addition into a phase gradient.

  This is in opposition to controlled rotations. It uses $n^2$ T gates, etc, etc
  """

Reply:
fixed